RecoParticleFlow/PFTracking/plugins/GoodSeedProducer.cc

```diff
@@ -222,11 +220,11 @@ GoodSeedProducer::produce(Event& iEvent, const EventSetup& iSetup)
 	  int ipteta=getBin(Tk[i].eta(),Tk[i].pt());
 	  int ibin=ipteta*9;
 	  
-	  float oPTOB=1.f/Tj[i].lastMeasurement().updatedState().globalMomentum().mag();
+	  float oPTOB=1.f/std::sqrt(Tk[i].innerMomentum().mag2()); // FIXME the original code was buggy should be outerMomentum...
```

Comment:
the current code produces identical results to the previous (verified with printout before migrating)
PTOB to me  sounds very much as P at TOB...